File: level 98/HW/mainhw98.js
function unique (s) {
    let ls = [];
    for (const i of s) {
        if (!ls.includes(i)){
            ls.push(i);
        }else{
            return false
        }
    }
    return true
}

console.log(unique([1,2,3,4,5]))


function matrixUnique (matrix) {
    let ls = [];
    for (const i of matrix) {
        if (unique(i)) {
            ls.push(i);
        }
    }
    return ls
}

File: level 98/HW/mainhw98.py
# def uniquematrix (matrix):
#     ls = []
#     for i in matrix:
#         for j in i:
#             if j not in ls:
#                 ls.append(j)
            
            
            
def unique (s):
    ls = []
    for i in s:
        if i not in ls:
            ls.append(i)
        else:
            return False
    return True

def matrix_unicue (matrix):
    ls = []
    for i in matrix:
            if unique(i):
                ls.append(i)
    return ls

print(matrix_unicue([[1,2,3],[5,6,7,7],[1,2,3,8,9,4]]))